versions-4.3.x.cfg

```diff
@@ -15,10 +15,9 @@ test-eggs =
 
 [versions]
 collective.js.bootstrap = 2.3.1.1
-plone.app.blocks = 3.1.0
-plone.app.drafts = 1.0
-plone.app.tiles = 1.1.0
-plone.tiles = 1.8.0
+plone.app.blocks = 4.0.6
+plone.app.tiles = 3.0.3
+plone.tiles = 2.0.0b2
 
 # for testing compatibility with newer versions of jQuery
 plone.app.jquery = 1.7.2
```

Comment:
Plone already [pin plone.app.jquery](https://github.com/plone/buildout.coredev/blob/4.3/versions.cfg#L166). I think it can be removed and treated [this](https://github.com/collective/collective.cover/blob/master/.travis.yml#L29) of another way.